metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -194,16 +145,19 @@ func (sink *StackdriverSink) ExportData(dataBatch *core.DataBatch) {
 			continue
 		}
 
+		// Hack used only with legacy resource type "gke_container". It is used to represent three
```

Comment:
`Hack used only with legacy resource type "gke_container"`

How is it represented in the logic? I don't see any mentions of the old resource type

Reply:
Clarified the comment a little bit, PTAL

Reply:
There's `sink.useOldResourceModel` in the condition, now I see. Sorry if I just missed it before